src/test/java/com/flair/bi/service/TestNotificationsGrpcService.java

```diff
@@ -154,4 +155,10 @@ public String disableTicketCreationRequest(Integer schedulerTaskLogId) {
 		// TODO Auto-generated method stub
 		return null;
 	}
+
+	@Override
```

Comment:
remove this test case from PR. there is separate PR created for this test case and can not be merged until proto buf version is not released